generated/nidcpower/nidcpower_service.cpp

```diff
@@ -11,6 +11,7 @@
 #include <iostream>
 #include <atomic>
 #include <vector>
+const auto kErrorReadBufferTooSmall = -200229;
```

Comment:
Should this go inside the namespace?

Do the drivers not already define this? Or they define the same value under different names?

Reply:
They use the same value under different names (and for some reason in this repo only `DAQmx.h` does, not sure why the other drivers don't...)

Yeah, I'll move it in the namespace, thanks!